app/views/admin/account_requests/_rejection_modal.html.erb

```diff
@@ -23,14 +23,3 @@
   </div>
 
 </div>
-
-<script type="module">
```

Comment:
Was this not being used? Or maybe the generalization in the index.html.erb takes care of it?

Reply:
I put it again! Fixing the missing specs I realized there is another flow that renders this modal bypassing the index.html, so its duplicated because that. Thx for the correction